Finished translation into Russian

buthed opened the pull request:
This PR Adds Russian translation to the project.

I'm releasing the below contribution under MIT license.

issue #1546

----------------------------------

**Tests** 

Found a few flaws:
Main menu does not fit text:

![image](https://user-images.githubusercontent.com/17142699/147685363-bc238281-5d43-4e7d-aca2-73581c12965c.png)


INBOX and OUTBOX fields are missing in the file strings.xml:

![image](https://user-images.githubusercontent.com/17142699/147685800-15ad076a-a46c-4c7a-842d-f41a2e7084f3.png)
![image](https://user-images.githubusercontent.com/17142699/147685808-3db2ad3e-8769-4e1f-b6cf-f62522f70f6e.png)

IvanPizhenko commented:
@DenBond7 I am in progress of reviewing this. Some corrections will be required.

tomholub commented:
@buthed thanks for the contribution!

IvanPizhenko commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,486 @@
+    <string name="pref_title_general">Общий</string>
+    <string name="get_access_to_gmail">Получите доступ к Gmail</string>
+    <string name="password_quality_good">хорошо</string>
+    <string name="password_quality_great">отлично</string>
```
отличное

IvanPizhenko commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,486 @@
+    <string name="forward_encrypted">Переслать с шифрованием</string>
+    <string name="pref_title_general">Общий</string>
+    <string name="get_access_to_gmail">Получите доступ к Gmail</string>
+    <string name="password_quality_good">хорошо</string>
```
хорошее

IvanPizhenko commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,486 @@
+    <string name="get_access_to_gmail">Получите доступ к Gmail</string>
+    <string name="password_quality_good">хорошо</string>
+    <string name="password_quality_great">отлично</string>
+    <string name="hide_original_message">Скрыть исходный контент</string>
```
исходное сообщение

IvanPizhenko commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,486 @@
+    <string name="password_quality_good">хорошо</string>
+    <string name="password_quality_great">отлично</string>
+    <string name="hide_original_message">Скрыть исходный контент</string>
+    <string name="pref_title_http_logs">Http логи</string>
```
Журналы Http

IvanPizhenko commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,486 @@
+    <string name="hide_original_message">Скрыть исходный контент</string>
+    <string name="pref_title_http_logs">Http логи</string>
+    <string name="have_not_any_other_key">У меня нет другого ключа</string>
+    <string name="pref_title_imap_debug">IMAP дебаг</string>
```
Отладка IMAP

IvanPizhenko commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,486 @@
+    <string name="import_missing_key">Импортировать отсутствующий ключ</string>
+    <string name="import_public_key">Импортировать открытый ключ</string>
+    <string name="incoming_mail_settings">Настройки входящей почты</string>
+    <string name="password_is_incorrect">Неверная ключ-фраза, попробуйте еще раз</string>
```
ключевая фраза

IvanPizhenko commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,486 @@
+    <string name="licence">Лицензия</string>
+    <string name="loading_backups">Загрузка резервных копий</string>
+    <string name="please_wait_while_information_about_recipients_will_be_updated">Загрузка информации о получателях, пожалуйста, подождите</string>
+    <string name="please_wait_while_backup_will_be_saved">Делаем резервную копию, подождите</string>
```
пожалуйста подождите

IvanPizhenko commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,486 @@
+    <string name="loading_backups">Загрузка резервных копий</string>
+    <string name="please_wait_while_information_about_recipients_will_be_updated">Загрузка информации о получателях, пожалуйста, подождите</string>
+    <string name="please_wait_while_backup_will_be_saved">Делаем резервную копию, подождите</string>
+    <string name="manage_notifications">Управлять уведомлениями</string>
```
Управление уведомлениями

IvanPizhenko commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,486 @@
+    <string name="no_public_key_recorded">Открытый ключ не записан. Ваши контакты могут не знать, что вы используете шифрование.</string>
+    <string name="notification_level">Уровень уведомления</string>
+    <string name="notifications">Уведомления</string>
+    <string name="passphrase_purged_from_memory">Ключ-фраза очищена из памяти</string>
```
Ключевая фраза

IvanPizhenko commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,486 @@
+    <string name="notification_level">Уровень уведомления</string>
+    <string name="notifications">Уведомления</string>
+    <string name="passphrase_purged_from_memory">Ключ-фраза очищена из памяти</string>
+    <string name="pass_phrase_changed">Ключ-фраза успешно изменена</string>
```
Ключевая фраза

IvanPizhenko commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,486 @@
+    <string name="passphrase_purged_from_memory">Ключ-фраза очищена из памяти</string>
+    <string name="pass_phrase_changed">Ключ-фраза успешно изменена</string>
+    <string name="please_login_again_to_continue">Пожалуйста, войдите снова, чтобы продолжить</string>
+    <string name="password_quality_poor">недостаточный</string>
```
плохое

IvanPizhenko commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,486 @@
+    <string name="password_quality_poor">недостаточный</string>
+    <string name="public_">Открытый</string>
+    <string name="private_">Закрытый</string>
+    <string name="processing_please_wait">Обработка, подождите&#8230;</string>
```
пожалуйста подождите

IvanPizhenko commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,486 @@
+    <string name="receive_notifications_for">Получать уведомления для:</string>
+    <string name="recipient_already_exist">Получатель %1$s уже существует</string>
+    <string name="to_receiver">Кому %1$s</string>
+    <string name="template_warning_max_total_attachments_size">Общий размер добавленных вложений не должен превышать         %1$s</string>
```
убрать лишние пробелы

IvanPizhenko commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,486 @@
+    <string name="use_public_key_from">Использовать открытый ключ из</string>
+    <string name="template_users">Пользователи: %1$s</string>
+    <string name="warning">"Предупреждение: "</string>
+    <string name="password_quality_weak">слабый</string>
```
слабое

IvanPizhenko commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,486 @@
+    <string name="template_users">Пользователи: %1$s</string>
+    <string name="warning">"Предупреждение: "</string>
+    <string name="password_quality_weak">слабый</string>
+    <string name="pref_title_write_api_logs_to_file">Записывать логи API в файл</string>
```
журналы

IvanPizhenko commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,486 @@
+    <string name="pref_title_write_api_logs_to_file">Записывать логи API в файл</string>
+    <string name="wrong_public_key_recorded">Записан неверный открытый ключ. Входящая зашифрованная электронная почта может быть недоступна для чтения.</string>
+    <string name="can_not_delete_sending_message">Вы не можете удалить сообщение, которое сейчас отправляется</string>
+    <string name="decrypt_error_current_key_cannot_open_message">Ваш текущий ключ не может открыть это сообщение. Если у вас есть другие ключи, вы должны импортировать их сейчас.</string>
```
вы должны -> вам следует
вы должны было бы "you must", а там "you should"  в оригинале

IvanPizhenko commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,486 @@
+    <string name="decrypt_error_current_key_cannot_open_message">Ваш текущий ключ не может открыть это сообщение. Если у вас есть другие ключи, вы должны импортировать их сейчас.</string>
+    <string name="your_key_backup">Ваша резервная копия FlowCrypt</string>
+    <string name="your_keys_cannot_open_this_message">Ваши ключи не могут открыть это сообщение</string>
+    <string name="pass_phrase_is_too_weak">Ваша ключ-фраза слишком слабая</string>
```
ключевая фраза

IvanPizhenko commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,486 @@
+    <string name="your_key_backup">Ваша резервная копия FlowCrypt</string>
+    <string name="your_keys_cannot_open_this_message">Ваши ключи не могут открыть это сообщение</string>
+    <string name="pass_phrase_is_too_weak">Ваша ключ-фраза слишком слабая</string>
+    <string name="passphrase_was_changed">Ваша ключ-фраза была изменена. Отныне новые резервные копии будут защищены заданной ключ-фразой.</string>
```
ключевая фраза

IvanPizhenko commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,486 @@
+    <string name="your_keys_cannot_open_this_message">Ваши ключи не могут открыть это сообщение</string>
+    <string name="pass_phrase_is_too_weak">Ваша ключ-фраза слишком слабая</string>
+    <string name="passphrase_was_changed">Ваша ключ-фраза была изменена. Отныне новые резервные копии будут защищены заданной ключ-фразой.</string>
+    <string name="different_pass_phrases">У ваших приватных ключей разные парольные фразы</string>
```
закрытых ключей

IvanPizhenko commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,486 @@
+    <string name="preparing">Подготовка&#8230;</string>
+    <string name="decrypt_error_please_write_me">Напишите нам по адресу %1$s, чтобы мы могли исправить это для вас.</string>
+    <string name="setup_app">Пожалуйста, настройте приложение, прежде чем использовать эту опцию</string>
+    <string name="select_stronger_pass_phrase">Пожалуйста, выберите более надежную ключ-фразу. Лучше всего сочетать от 4 до 5 необычных слов.</string>
```
ключевую фразу

IvanPizhenko commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,486 @@
+    <string name="setup_app">Пожалуйста, настройте приложение, прежде чем использовать эту опцию</string>
+    <string name="select_stronger_pass_phrase">Пожалуйста, выберите более надежную ключ-фразу. Лучше всего сочетать от 4 до 5 необычных слов.</string>
+    <string name="reconnect_your_account">Пожалуйста, повторно подключите свою учетную запись, чтобы продолжить использование %1$s с %2$s</string>
+    <string name="provide_passphrase_to_decrypt_file">Пожалуйста, укажите ключ-фразу для расшифровки этого файла</string>
```
ключевую фразу

IvanPizhenko commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,486 @@
+    <string name="reconnect_your_account">Пожалуйста, повторно подключите свою учетную запись, чтобы продолжить использование %1$s с %2$s</string>
+    <string name="provide_passphrase_to_decrypt_file">Пожалуйста, укажите ключ-фразу для расшифровки этого файла</string>
+    <string name="failed_load_message_from_email_server">Не удалось загрузить сообщение с почтового сервера</string>
+    <string name="failed_please_try_again_later">Не удалось. Пожалуйста, повторите попытку позже</string>
```
Не удалось завершить операцию. 
(просто не удалось звучит как-то странно)
возможно английский вариант тоже надо исправить

IvanPizhenko commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,486 @@
+    <string name="failed_load_message_from_email_server">Не удалось загрузить сообщение с почтового сервера</string>
+    <string name="failed_please_try_again_later">Не удалось. Пожалуйста, повторите попытку позже</string>
+    <string name="experimental_settings">Экспериментальные настройки</string>
+    <string name="expand_an_account_managment">Расширьте управление аккаунтом</string>
```
Развернуть управление аккаунтом

IvanPizhenko commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,486 @@
+    <string name="crash_the_app">Сбой приложения</string>
+    <string name="all_labels">Все вкладки</string>
+    <string name="security_and_privacy">Безопасность и конфиденциальность</string>
+    <string name="legal">Политики</string>
```
Юридическая информация

IvanPizhenko commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,486 @@
+    <string name="all_labels">Все вкладки</string>
+    <string name="security_and_privacy">Безопасность и конфиденциальность</string>
+    <string name="legal">Политики</string>
+    <string name="experimental">Дополнительные</string>
```
Экспериментальные

IvanPizhenko commented on FlowCrypt/src/main/res/values/strings.xml:
```diff
@@ -556,4 +556,4 @@
     <string name="not_signed">not signed</string>
     <string name="mixed_signed">mixed signed</string>
-</resources>
+</resources>
```
should not have been changed, please revert this file.

IvanPizhenko reviewed (changes requested):
@buthed @DenBond7  Some changes required. Please see my comments.

IvanPizhenko commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,486 @@
+    <string name="no_public_key_recorded">Открытый ключ не записан. Ваши контакты могут не знать, что вы используете шифрование.</string>
+    <string name="notification_level">Уровень уведомления</string>
+    <string name="notifications">Уведомления</string>
+    <string name="passphrase_purged_from_memory">Ключевая фраза очищена из памяти</string>
```
удалена из памяти

IvanPizhenko commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,486 @@
+    <string name="processing_please_wait">Обработка, пожалуйста подождите&#8230;</string>
+    <string name="public_key_details">Подробная информация об открытом ключе</string>
+    <string name="template_message_part_public_key_owner">Открытый ключ для %1$s</string>
+    <string name="public_key_below_is_safe_to_share">"Можно делиться открытыми ключами безопасно "</string>
```
лишние кавычки, их нет в оригинале

IvanPizhenko commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,486 @@
+    <string name="template_warning_max_total_attachments_size">Общий размер добавленных вложений не должен превышать %1$s</string>
+    <string name="use_public_key_from">Использовать открытый ключ из</string>
+    <string name="template_users">Пользователи: %1$s</string>
+    <string name="warning">"Предупреждение: "</string>
```
убрать кавычки

IvanPizhenko commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,486 @@
+    <string name="pref_title_write_api_logs_to_file">Записывать журналы API в файл</string>
+    <string name="wrong_public_key_recorded">Записан неверный открытый ключ. Входящая зашифрованная электронная почта может быть недоступна для чтения.</string>
+    <string name="can_not_delete_sending_message">Вы не можете удалить сообщение, которое сейчас отправляется</string>
+    <string name="decrypt_error_current_key_cannot_open_message">Ваш текущий ключ не может открыть это сообщение. Если у вас есть другие ключи, вам следует импортировать их сейчас.</string>
```
думаю, что лучше это перефразировать:
Если у вас есть другие ключи, импортируйте их.

IvanPizhenko commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,486 @@
+    <string name="your_keys_cannot_open_this_message">Ваши ключи не могут открыть это сообщение</string>
+    <string name="pass_phrase_is_too_weak">Ваша ключевая фраза слишком слабая</string>
+    <string name="passphrase_was_changed">Ваша ключевая фраза была изменена. Отныне новые резервные копии будут защищены заданной ключ-фразой.</string>
+    <string name="different_pass_phrases">У ваших закрытых ключей разные парольные фразы</string>
```
различные ключевые фразы

IvanPizhenko commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,486 @@
+    <string name="pass_phrase_is_too_weak">Ваша ключевая фраза слишком слабая</string>
+    <string name="passphrase_was_changed">Ваша ключевая фраза была изменена. Отныне новые резервные копии будут защищены заданной ключ-фразой.</string>
+    <string name="different_pass_phrases">У ваших закрытых ключей разные парольные фразы</string>
+    <string name="primary_address_not_defined">Основной адрес не определен</string>
```
не задан

IvanPizhenko commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,486 @@
+    <string name="preparing">Подготовка&#8230;</string>
+    <string name="decrypt_error_please_write_me">Напишите нам по адресу %1$s, чтобы мы могли исправить это для вас.</string>
+    <string name="setup_app">Пожалуйста, настройте приложение, прежде чем использовать эту опцию</string>
+    <string name="select_stronger_pass_phrase">Пожалуйста, выберите более надежную ключевую фразу. Лучше всего сочетать от 4 до 5 необычных слов.</string>
```
Лучше всего использовать сочетание 4 или 5 необычных слов.

IvanPizhenko commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,486 @@
+    <string name="decrypt_error_please_write_me">Напишите нам по адресу %1$s, чтобы мы могли исправить это для вас.</string>
+    <string name="setup_app">Пожалуйста, настройте приложение, прежде чем использовать эту опцию</string>
+    <string name="select_stronger_pass_phrase">Пожалуйста, выберите более надежную ключевую фразу. Лучше всего сочетать от 4 до 5 необычных слов.</string>
+    <string name="reconnect_your_account">Пожалуйста, повторно подключите свою учетную запись, чтобы продолжить использование %1$s с %2$s</string>
```
missing EOLs

IvanPizhenko commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,486 @@
+    <string name="failed_please_try_again_later">Не удалось завершить операцию. Пожалуйста, повторите попытку позже</string>
+    <string name="experimental_settings">Экспериментальные настройки</string>
+    <string name="expand_an_account_managment">Развернуть управление аккаунтом</string>
+    <string name="evaluating">Оценка…</string>
```
В данном случае, исходя из того где оно используется, лучше перевести как
Обработка...

IvanPizhenko commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,486 @@
+    <string name="backup_was_not_sent">Ошибка резервного копирования, попробуйте еще раз</string>
+    <string name="found_not_fully_encrypted_key">Ошибка. Ключ %1$s зашифрован не полностью</string>
+    <string name="found_not_fully_decrypted_key_ask_admin">Ошибка. Ключ %1$s не полностью расшифрован. Обратитесь к системному администратору или в службу поддержки</string>
+    <string name="crash_the_app">Сбой приложения</string>
```
Искусственный сбой приложения

IvanPizhenko reviewed (changes requested):
@DenBond7 @buthed I've done a more careful review after last updates, and found more points that needs to be fixed. @buthed please make corrections and tag me and @DenBond7 when that is ready. Please pay special attention to missing newlines (`\n`) and extra quotes in some translations.

buthed commented:
@IvanPizhenko @DenBond7 
I have corrected all the points you mentioned.
Checked the code again and added any missing (\ n).
Also found that ru file was missing Plurals lines.

PS.
@IvanPizhenko sorry for the many mistakes and thanks for your careful review.

IvanPizhenko commented:
@buthed please see a bit more notes right in the your last commit (not a lot). when you address them, I think we are ready to merge.

IvanPizhenko commented:
@buthed I have one more question - the folder where translations are placed is called `values-ru-rRU`. Please note that extra `r` before `RU`. Is that correct? Shouldn't it be w/o that `r`? Also @DenBond7 what do you think of this?

IvanPizhenko commented:
@buthed ok, all translation notes are addressed, still my question about the folder naming

buthed commented:
@IvanPizhenko Done.
I do not know.
The folder was created automatically.

IvanPizhenko commented:
> @IvanPizhenko Done. I do not know. The folder was created automatically.

ok, then let's wait for @DenBond7 comment on this.

DenBond7 commented:
> @buthed I have one more question - the folder where translations are placed is called `values-ru-rRU`. Please note that extra `r` before `RU`. Is that correct? Shouldn't it be w/o that `r`? Also @DenBond7 what do you think of this?

In our case it's not correct
Please look at the following examples

Here we specify a new locale for Russian at all
![image](https://user-images.githubusercontent.com/2863246/148183916-e871378a-5215-4786-912b-0a86d4ea38ff.png)

@buthed has added a version that looks like this one

![image](https://user-images.githubusercontent.com/2863246/148184109-98cef1d6-aca3-4076-90b2-94283c00c6a5.png)

In this case the translation will be available only for Russian locale where country == Russia (Ukraine, Belarus and so on will be rejected)

P.S. Sorry for the delayed reply. Too many other tasks.

DenBond7 commented:
Before process with this I should review the current strings and exclude some from translation.

IvanPizhenko commented:
@buthed Den has marked some string as non-translatable, and this PR #1638 is merged now. Please sync your branch with upstream master and then exclude from your translations all strings marked as non-translatable in the original string list.
Please also take into account Den's comment about directory name.

DenBond7 commented:
@buthed @IvanPizhenko That's not all. I'm on the reviewing process.

DenBond7 commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,487 @@
+    <string name="security_type">Тип протокола безопасности</string>
+    <string name="require_custom_sign_in">Использовать другие учетные данные для SMTP</string>
+    <string name="template_email_already_added">Сообщение  \"%1$s\" уже добавлено</string>
+    <string name="keys_have_different_pass_phrase">У ключей другая ключева фраза</string>
```
Ключи имеют разную ключевую фразу

DenBond7 commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,487 @@
+    <string name="pref_summary_disabled">Отключен</string>
+    <string name="action_dev_settings">Настройки разработчика</string>
+    <string name="details">Подробности</string>
+    <string name="delete_quoted_text">Удалить выделенный текст</string>
```
цитируемый

DenBond7 commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,487 @@
+    <string name="open_side_menu_and_do_logout">Откройте боковое меню и выйдите из системы, чтобы удалить %1$s</string>
+    <string name="loading_oauth_server_configuration">Загрузка конфигурации сервера OAuth&#8230;</string>
+    <string name="loading_account_details">Загрузка информации об учетной записи &#8230;</string>
+    <string name="provider_faq">\n\nЧасто задаваемые вопросы о провайдере: %1$s</string>
```
FAQ от провайдера: %1$s

DenBond7 commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,487 @@
+    <string name="security_and_privacy">Безопасность и конфиденциальность</string>
+    <string name="legal">Юридическая информация</string>
+    <string name="experimental">Экспериментальные</string>
+    <string name="loading_labels">Загрузка вкладок</string>
```
Загрузка ярлыков

DenBond7 commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,487 @@
+    <string name="could_not_decrypt_message_due_to_error">Не удалось расшифровать это сообщение из-за ошибки: \n\n %1$s</string>
+    <string name="could_not_create_no_key_available">Не удалось создать: ключ недоступен</string>
+    <string name="could_not_create">Не удалось создать: нажмите, чтобы увидеть подробности</string>
+    <string name="could_not_check_pass_phrase">Не удалось проверить парольную ключевую фразу</string>
```
Не удалось проверить ключевую фразу

DenBond7 commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,487 @@
+    <string name="copy_from_other_contact">Скопировать из другого контакта</string>
+    <string name="copy_to_clipboard">Скопировать в буфер обмена</string>
+    <string name="could_not_import_data">Не удалось импортировать данные</string>
+    <string name="forget_passphrase">Забыл ключевую фразу</string>
```
Забыть ключевую фразу

DenBond7 commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,487 @@
+    <string name="template_fingerprint"><![CDATA[Отпечаток: <font color=\'#B7B7B7\'>%1$s</font>]]></string>
+    <string name="password_quality_subtext">(время взлома:</string>
+    <string name="there_are_no_backups_on_this_account"><b>В этой учетной записи нет резервных копий</b>\n\n  Если вы потеряете свое устройство или оно перестанет работать, ваша зашифрованная электронная почта станет нечитаемой.</string>
+    <string name="active_passphrase_session">Активная сесссия ключевой фразы</string>
```
сессия

DenBond7 commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,487 @@
+    <string name="msg_contains_not_valid_block">Это сообщение содержит некорректный блок с типом = %1$s.\n\n%2$s</string>
+    <string name="msg_contains_not_valid_pub_key">Это сообщение содержит недействительный открытый ключ.\n\n%1$s</string>
+    <string name="this_message_will_not_be_encrypted">Это сообщение не будет зашифровано</string>
+    <string name="key_sha1_warning_msg">Этот ключ OpenPGP не поддерживается с сообщением: Подходящие подписи для ключа не найдены. Это может быть из-за того, что отсутствуют внутренние подписи, или подписи сделаны неправильно, или он может использовать устаревшие алгоритмы, которые больше не являются безопасными и больше не поддерживаются (например, SHA-1).</string>
```
Этот ключ OpenPGP не поддерживается. Подробности: подходящие подписи для ключа не найдены. Это может быть из-за того, что отсутствуют внутренние подписи, или подписи сделаны неправильно, или он может использовать устаревшие алгоритмы, которые больше не являются безопасными и больше не поддерживаются (например, SHA-1).

DenBond7 commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,487 @@
+    <string name="pass_phrase_is_too_weak">Ваша ключевая фраза слишком слабая</string>
+    <string name="passphrase_was_changed">Ваша ключевая фраза была изменена. Отныне новые резервные копии будут защищены заданной ключ-фразой.</string>
+    <string name="different_pass_phrases">У ваших закрытых ключей различные ключевые фразы</string>
+    <string name="primary_address_not_defined">Основной адрес не задан</string>
```
Основной адрес не определен

DenBond7 commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,487 @@
+    <string name="you_trying_replace_pub_key_with_older_version">Вы пытаетесь заменить открытый ключ на более старую версию</string>
+    <string name="cannot_be_used_for_encryption">Предупреждение: нельзя использовать для шифрования</string>
+    <string name="warning_about_manual_import">Импортирование вручную открытых ключей, полученных по электронной почте, может быть опасным. Свяжитесь с отправителем, чтобы убедиться, что отпечаток ключа совпадает.</string>
+    <string name="manually_importing_public_keys_can_be_dangerous">Импортирование вручную открытых ключей вручную может быть опасным</string>
```
Импортирование открытых ключей вручную может быть опасным

DenBond7 commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,487 @@
+    <string name="empty_trash">Очистить корзину</string>
+    <string name="pref_title_enable_acra">Включить ACRA(App Crash Reports)</string>
+    <string name="pref_summary_enabled">Включено</string>
+    <string name="encrypted">зашифрованный</string>
```
зашифровано

DenBond7 commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,487 @@
+    <string name="load_account">Загрузить учетную запись</string>
+    <string name="no_backups_found">В этой учетной записи не найдено резервных копий</string>
+    <string name="pref_title_http">Http</string>
+    <string name="signed">подписан</string>
```
подписано

DenBond7 commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,487 @@
+    <string name="warning_one_of_recipients_has_revoked_pub_key">Один или несколько ваших получателей отозвали открытые ключи (отмечены красным).\n\nПопросите их прислать вам новый открытый ключ. Если это корпоративная установка, обратитесь к системному администратору.</string>
+    <string name="warning_one_of_recipients_has_expired_pub_key">У одного или нескольких ваших получателей истек срок действия открытых ключей (отмечен оранжевым).\n\nПопросите их прислать вам обновленный открытый ключ. Если это корпоративная установка, обратитесь к системному администратору.</string>
+    <string name="partially_encrypted_private_key_error_msg">Один из предоставленных закрытых ключей - это частично зашифрованный закрытый ключ (некоторые пакеты зашифрованы, некоторые нет). Такие ключи не поддерживаются. Пожалуйста, импортируйте только полностью зашифрованные ключи с одной и той же парольной фразой для всех секретных пакетов.</string>
+    <string name="not_signed">не подписан</string>
```
не подписано

buthed commented on FlowCrypt/src/main/res/values-ru-rRU/strings.xml:
```diff
@@ -0,0 +1,487 @@
+    <string name="info">Информация</string>
+    <string name="inform_sender">Сообщить отправителю</string>
+    <string name="pref_title_java_mail">JavaMail</string>
+    <string name="key_successfully_imported">Ключ(и) успешно импортированы</string>
```
Changed translation without using plurals.

buthed commented:
> 
So far i found one problem related to word length^
![image](https://user-images.githubusercontent.com/17142699/148292797-5d834c01-4390-4167-86b1-23ccce4653a1.png)

buthed commented:
> > @buthed I have one more question - the folder where translations are placed is called `values-ru-rRU`. Please note that extra `r` before `RU`. Is that correct? Shouldn't it be w/o that `r`? Also @DenBond7 what do you think of this?
> 
> In our case it's not correct Please look at the following examples
> 
> Here we specify a new locale for Russian at all ![image](https://user-images.githubusercontent.com/2863246/148183916-e871378a-5215-4786-912b-0a86d4ea38ff.png)
> 
> @buthed has added a version that looks like this one
> 
> ![image](https://user-images.githubusercontent.com/2863246/148184109-98cef1d6-aca3-4076-90b2-94283c00c6a5.png)
> 
> In this case the translation will be available only for Russian locale where country == Russia (Ukraine, Belarus and so on will be rejected)
> 
> P.S. Sorry for the delayed reply. Too many other tasks.

You're right. Rebuilt to directory for all regions.

IvanPizhenko commented:
@buthed 3 more things to fix after reviewing your last commit.

buthed commented:
@IvanPizhenko Done

IvanPizhenko commented:
> @IvanPizhenko Done

@buthed Thank you! So we will wait now for the final word from the @DenBond7

DenBond7 commented on FlowCrypt/src/main/res/values-ru/strings.xml:
```diff
@@ -0,0 +1,467 @@
+    <string name="this_message_will_not_be_encrypted">Это сообщение не будет зашифровано</string>
+    <string name="key_sha1_warning_msg">Этот ключ OpenPGP не поддерживается. Подробности: подходящие подписи для ключа не найдены. Это может быть из-за того, что отсутствуют внутренние подписи, или подписи сделаны неправильно, или он может использовать устаревшие алгоритмы, которые больше не являются безопасными и больше не поддерживаются (например, SHA-1).</string>
+    <string name="backup_as_email_hint">Это создаст резервную копию закрытых ключей в Вашем почтовом ящике.  Степень безопасности зависит от силы Вашего пароля, который должен быть очень длинным, и трудноугадываемым. Лучший вариант для большинства пользователей.</string>
+    <string name="backup_as_download_hint">Это сохранит Ваши ключи в виде файла. Очень безопасный вариант, если Ваше устройство зашифровано или если Вы можете хранить этот файл в безопасности. Если Вы потеряете и телефон, и файл, ваше зашифрованное электронное письмо станет нечитаемым, поэтому вам необходимо сохранить этот файл в надежном месте.</string>
```
Ваше зашифрованное
поэтому Вам

DenBond7 commented on FlowCrypt/src/main/res/values-ru/strings.xml:
```diff
@@ -0,0 +1,467 @@
+    <string name="passphrase_purged_from_memory">Ключевая фраза удалена из памяти</string>
+    <string name="pass_phrase_changed">Ключевая фраза успешно изменена</string>
+    <string name="please_login_again_to_continue">Пожалуйста, войдите снова, чтобы продолжить</string>
+    <string name="password_quality_poor">плохое</string>
```
плохая(ключевая фраза)

DenBond7 reviewed (changes requested):
Please fix a few comments. Overall, it already looks good! I will approve it after fixing.

IvanPizhenko commented:
@DenBond7 CI did not run here and this does not let merging this PR normally. I believe it's because this PR comes from the external user which does not have permissions for our CI. Can you please forcemerge it? Since both of us have already approved it.
